src/main/java/com/azure/cosmos/examples/patch/sync/SamplePatchQuickstart.java

```diff
@@ -0,0 +1,490 @@
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.CosmosException;
+import com.azure.cosmos.CosmosPatchOperations;
+import com.azure.cosmos.TransactionalBatch;
+import com.azure.cosmos.TransactionalBatchResponse;
+import com.azure.cosmos.examples.common.AccountSettings;
+import com.azure.cosmos.examples.common.Families;
+import com.azure.cosmos.examples.common.Family;
+import com.azure.cosmos.examples.common.Child;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerResponse;
+import com.azure.cosmos.models.CosmosDatabaseResponse;
+import com.azure.cosmos.models.CosmosItemRequestOptions;
+import com.azure.cosmos.models.CosmosItemResponse;
+import com.azure.cosmos.models.CosmosPatchItemRequestOptions;
+import com.azure.cosmos.models.CosmosQueryRequestOptions;
+import com.azure.cosmos.models.PartitionKey;
+import com.azure.cosmos.models.ThroughputProperties;
+import com.azure.cosmos.util.CosmosPagedIterable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SamplePatchQuickstart {
+
+    private CosmosClient client;
+
+    private final String databaseName = "AzureSampleFamilyDB";
+    private final String containerName = "FamilyContainer";
+
+    private CosmosDatabase database;
+    private CosmosContainer container;
+
+    protected static Logger logger = LoggerFactory.getLogger(SamplePatchQuickstart.class);
+
+    public void close() {
+        client.close();
+    }
+
+    /**
+     * Run a Hello CosmosDB console application.
+     * <p>
+     * This is a simple sample application intended to demonstrate Create, Read,
+     * Update, Delete (CRUD) operations with Azure Cosmos DB Java SDK, as applied to
+     * databases, containers and items. This sample will 1. Create synchronous
+     * client, database and container instances 2. Create several items 3. Upsert
+     * one of the items 4. Perform a query over the items 5. Delete an item 6.
+     * Delete the Cosmos DB database and container resources and close the client. *
+     */
+    // <Main>
+    public static void main(String[] args) {
+        SamplePatchQuickstart p = new SamplePatchQuickstart();
+
+        try {
+            logger.info("Starting SYNC main");
+            p.getStartedDemo();
+            logger.info("Demo complete, please hold while resources are released");
+        } catch (Exception e) {
+            e.printStackTrace();
+            logger.error(String.format("Cosmos getStarted failed with %s", e));
+        } finally {
+            logger.info("Closing the client");
+            p.shutdown();
+        }
+    }
+
+    // </Main>
+
+    private void getStartedDemo() throws Exception {
```

Comment:
maybe create 2 functions inside this function. 
one function as builder function that will do all the setup or preprocessing required(database setup n stuff)
second function is where we put patch stuff.

Reply:
I think it's fine the way it is - just from the point of view of maintaining consistency with other examples